tests/script-change-testcase-return-type.php

```diff
@@ -4,5 +4,22 @@
     exit(0);
 }
 
-file_put_contents('tests/BaseTestCase.php', str_replace('public function setUp()', 'public function setUp(): void', file_get_contents('tests/BaseTestCase.php')));
-file_put_contents('tests/BaseTestCase.php', str_replace('public function tearDown()', 'public function tearDown(): void', file_get_contents('tests/BaseTestCase.php')));
+require_once './vendor/autoload.php';
+
+use Symfony\Component\Finder\Finder;
+
+$finder = new Finder();
+
+$finder->files()
+    ->ignoreVCS(false)
+    ->in(__DIR__)
+    ->name('*.php')
+    ->notName('script-change-testcase-return-type.php')->contains('use Orchestra\Testbench\TestCase;');
+
+foreach ($finder as $file) {
+    $absoluteFilePath = $file->getRealPath();
+    $fileNameWithExtension = $file->getRelativePathname();
+
+    file_put_contents($absoluteFilePath, str_replace('public function setUp()', 'public function setUp(): void', file_get_contents($absoluteFilePath)));
+    file_put_contents($absoluteFilePath, str_replace('public function tearDown()', 'public function tearDown(): void', file_get_contents($absoluteFilePath)));
```

Comment:
These lines can be simplified with something like this:
```php
$contents = file_get_contents($absoluteFilePath);
$contents = str_replace(
    ['public function setUp()', 'public function tearDown()'],
    ['public function setUp(): void', 'public function tearDown(): void'],
    $contents
);

file_put_contents($absoluteFilePath, $contents);
```

You should see this approach when you pull changes from the `6.x` branch.

Reply:
oh, I didn't know the array support for `str_replace`. I will change that